lib/brains/management/commands/braintree_config.py

```diff
@@ -0,0 +1,108 @@
+from decimal import Decimal
+from optparse import make_option
+
+from django.conf import settings
+from django.core.management.base import BaseCommand, CommandError
+from django.utils import importlib
+
+from lib.brains.client import get_client
+from lib.sellers.models import Seller, SellerProduct
+from solitude.logger import getLogger
+
+
+log = getLogger('s.brains.management')
+client = get_client()
+
+
+def import_file(path):
+    try:
+        res = importlib.import_module(path)
+    except ImportError:
+        raise CommandError('Could not import file {0}'.format(path))
+    try:
+        products = getattr(res, 'products')
+    except AttributeError:
+        raise CommandError('Cannot find "products" in file {0}'.format(path))
+    return products
+
+
+def get_or_create_seller(uuid):
+    """
+    Create a seller in solitude for the product.
+    """
+    # We'll suffix with the BRAINTREE_PUBLIC_ID so that the seller will
+    # change if you change ids, or servers.
```

Comment:
Ok so then its not a uuid and adding in the braintree_public_key should make it a uuid. I'm trying to ensure that if you flip your local solitude accounts around in development all the data doesn't get attached to the wrong rows. For example: do some work in account X, flip to account Y and you wouldn't want to start attaching sales to the seller created in account X.